code/cutout.py

```diff
@@ -0,0 +1,115 @@
+# function to extract cutout image
+import math
+from astropy.nddata import Cutout2D
+import astropy.io.fits as fits
+from astropy.wcs import WCS
+
+from exceptions import CutoutError
+
+
+def extract_pair(ra, dec, *, img_pair, cutout_width, mosaic_pix_scale):
```

Comment:
We don't necessarily need the extra function. There is a bit of logic in there that figures out whether to pull the background cutout from the science image or a separate file, so I put it in a function thinking it would be easier to use that way. But the logic ended up being really simple, so in the end it's probably fine to go in the notebook if folks think that's better.

Reply:
It's nitpicking for this use case (we can keep both), but in general I'm usually on the side of a simpler API. E.g having one function that two very similar ones. Here, in these use cases I think we can force the user to provide the "pairs", or to have the bkg input optional, rather than to provide both `extract` and `extract_pair`.